tests/behat/graph_helper.php

```diff
@@ -248,10 +257,56 @@ public function send_share_invitation(string $user, string $sharetype, string $s
         ];
         $itemid = $this->get_resource_id($user, $resource, $space);
         $client = $this->get_client($user);
-        return $client->request(
+        $response = $client->request(
             'POST',
             "/graph/v1beta1/drives/$spaceid/items/$itemid/invite",
             json_encode($body)
         );
+        $this->lastcreatedshare = $response;
```

Comment:
if `$this->lastcreatedshare` should only hold the id of the share, why putting the complete response into it?